conf-cc

```diff
@@ -1,3 +1,3 @@
-cc -O2
+cc -O2 -Wimplicit-fallthrough
```

Comment:
I like this idea to incrementally enable specific warnings in `conf-cc` (all builds for all users) when we’re pretty sure we’ve thoroughly handled those warnings. Couldn’t we also enable `-Werror=implicit-fallthrough` here? (I might be making the same category of error as I was early in #189, but I don’t think so.)

Reply:
Yes we should. And I should do it in the CI, not in the default file for everyone.

Reply:
-Werror= doesn't work, there are other cases that still fallthrough.